projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/__mocks__/index.ts

```diff
@@ -0,0 +1,130 @@
+import { PartyMessages } from '../domain';
+
+export const partyMessagesMockData: PartyMessages[] = [
+  {
+    partyName: 'John Smith - Appellant',
+    roleOnCase: null,
+    partyMessages: [
+      {
+        id: 'case-message-001',
+        subject: 'Review attached document',
+        body: 'Please review attached document and advise if hearing should proceed?',
+        attachments: [],
+        isHearingRelated: true,
+        hearingDate: new Date(2023, 0, 10),
+        createdOn: new Date(2023, 0, 3),
+        createdBy: '1111-1111-1111-1111'
+      },
+      {
+        id: 'case-message-002',
+        subject: 'Games',
+        body: 'Can I play games in my phone when my solicitor is talking?',
+        attachments: [],
+        isHearingRelated: true,
+        hearingDate: new Date(2023, 0, 10),
+        createdOn: new Date(2023, 0, 3),
+        createdBy: '1111-1111-1111-1111'
+      },
+      {
+        id: 'case-message-003',
+        subject: 'Games',
+        body: 'Using mobile phone is strictly prohibited in the court room.',
+        attachments: [],
+        isHearingRelated: true,
+        hearingDate: new Date(2023, 0, 10),
+        createdOn: new Date(2023, 0, 4),
+        createdBy: '2222-2222-2222-2222',
+        parentId: 'case-message-002'
+      },
+      {
+        id: 'case-message-004',
+        subject: 'Games',
+        body: 'In addition to my previous message, the mobile phones must be switched off inside the premises.',
+        attachments: [],
+        isHearingRelated: true,
+        hearingDate: new Date(2023, 0, 10),
+        createdOn: new Date(2023, 0, 4),
+        createdBy: '2222-2222-2222-2222',
+        parentId: 'case-message-002'
```

Comment:
You cannot have multiple responses to 1 query.

Reply:
I have now updated the mock data.